setup.py

```diff
@@ -33,12 +33,14 @@
     'tutorial': ['jupyter', 'matplotlib'],
 }
 
+GMS_EXT = ['.gms', '.opt', '.md']
+
 
 def all_subdirs(path, strip=None):
     paths = []
-    for root, dirnames, filenames in os.walk(path):
-        for dirname in dirnames:
-            paths.append(os.path.join(root, dirname, '*'))
+    for root, dirnames, files in os.walk(path):
+        for f in [_f for _f in files if os.path.splitext(_f)[1] in GMS_EXT]:
```

Comment:
and here since you only use `GMS_EXT` here, I would just make it a list in the function.

Reply:
done